Gems/Multiplayer/Code/Source/Debug/MultiplayerConnectionViewportMessageSystemComponent.h

```diff
@@ -24,11 +24,13 @@ namespace Multiplayer
         , MultiplayerEditorServerNotificationBus::Handler
     {
     public:
+        static constexpr int MaxMessageLength = 256;
         static constexpr char CenterViewportDebugTitle[] = "Multiplayer Editor";
         static constexpr char ClientStatusTitle[] = "Multiplayer Client Status:";
         static constexpr char OnServerLaunchedMessage[] = "(1/3) Launching server...";
         static constexpr char OnServerLaunchFailMessage[] = "(1/3) Could not launch editor server.\nSee console for more info.";
-        static constexpr char OnEditorConnectionAttemptMessage[] = "(2/3) Editor tcp connection attempt #%i.";
+        static constexpr char OnEditorConnectionAttemptMessage[] = "(2/3) Editor TCP connection attempt #%i.";
+        static constexpr char OnAllConnectionAttemptsFailedMessage[] = "(2/3) All connection attempts failed!\nConnection limit is defined by 'editorsv_max_connection_attempts'\nPlease exit play mode and try again.";  
```

Comment:
I would keep this simpler - "Failed to connect to server after x attempts. Please exist play mode and try again"

Changing the `editorsv_max_connection_attempts` will not really help here in most cases AFAIK so would leave that to docs as an "advanced setting"

Reply:
Perfect. Fixed :)